lib/commentOnClosedIssue.js

```diff
@@ -116,4 +116,4 @@ function statusCodeIsOk(response) {
     return false;
 }
 
-commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com[^\s.,:]*/ig;
+commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com[^\s.,:)\\]*/ig;
```

Comment:
To "capture" the link.

When concierge posts to the issue, it posts the links that it found. This functionality could be removed, I guess.